src/components/Sum.jsx

```diff
@@ -1 +1,11 @@
 // export function Sum() {}
+import React from 'react';
+
+export const Sum = ({ a, b }) => (
+  <p>{`Sum of ${a} and ${b} is ${[a + b]}`}</p>
+);
+
+Sum.defaultProps = {
+  a: 0,
+  b: 0,
+};
```

Comment:
Maybe its better here to put them straight to the arguments.

Reply:
Done!